# Error Codes
int32 OBSTACLE_DETECTED_ERROR = 101
int32 MOVE_TO_USER_ERROR      = 102

# Error Message
int32 error
string info
